src/hotspot/share/prims/jvmtiThreadState.cpp

```diff
@@ -210,15 +210,17 @@ JvmtiThreadState::periodic_clean_up() {
   }
 }
 
-/* Virtual Threads Mount Transition (VTMT) mechanism */
+//
+// Virtual Threads Mount Transition (VTMT) mechanism
+//
 
-// VTMT can not be disabled while this counter is positive
+// VTMT cannot be disabled while this counter is positive
```

Comment:
A dot at the end is also needed to follow your comment style.

Reply:
Fixed.